modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc

```diff
@@ -43,10 +43,16 @@ You can use any of the following options for authentication and authorization wh
 
 ** Using <<create-api-ASM-CLI, user credentials>> (username and password)
 +
-This option supports only those users that are managed by Anypoint Platform.
+This option supports only those users that are:
+
+*** Managed by Anypoint Platform.
+*** Not configured with Multi-Factor authentication.
 ** Using <<create-api-ASM-CLI-ConnAps, Connected Apps credentials>> 
 +
-(Recommended) This option supports users regardless of where they are managed--Anypoint Platform or an external IdP.
+(Recommended) This option supports users irrespective of
```

Comment:
It feels odd to split the prepositional phrase across intro & bullets, and it would be nice if we had parallel construction with line 46, so, something like "This option supports users:

* Managed by Anypoint Platform or an external IdP.
* Whether or not they have enabled multi-factor authentication.

Reply:
Done! Thanks!